src/components/Test_Auctions.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import { Query } from 'react-apollo'
```

Comment:
In my opinion, is better not to have apollo logic in the components and pages. 
There should be an abstraction that returns the interfaces, so the UI is not so coupled to the datasource.
Note that is not 100% sure we'll be using GraphQL, and there's an open PR were it's being discussed to make a simplifications and consume just some basic smart contract data.

I guess this is just to test the data fetching, but maybe you can make a proposal on how to abstract this data fetching liberating the UI of this logics, and allowing different components to share queries and SC interactions.

Reply:
that's the route we will take, but as you said, for the sake of not over-complicating this is just an example or remote data fetching and it's presentation 

we will likely use withAccountsQL HOC components

Reply:
Okay, it can be done in a separate PR.